File: Drivers/FSCC Drivers/windows/lib/fscc/c/examples/rx-multiple.c
#include <fscc.h> /* fscc_* */

int main(void)
{
    fscc_handle h;
    unsigned status;

    fscc_connect(0, &h);

    fscc_get_rx_multiple(h, &status);

    fscc_enable_rx_multiple(h);
    fscc_disable_rx_multiple(h);

    fscc_disconnect(h);

    return 0;
}

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/rx-multiple.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);

        var status = p.RxMultiple;
        
        p.RxMultiple = true;
        p.RxMultiple = false;

        return 0;
    }
}